src/inspector/Inspector.zig

```diff
@@ -14,6 +14,7 @@ const input = @import("../input.zig");
 const renderer = @import("../renderer.zig");
 const terminal = @import("../terminal/main.zig");
 const inspector = @import("main.zig");
+const utils = @import("utils.zig");
```

Comment:
@mitchellh  I wasn't too sure about using utils here. Do you have any suggestions on the naming and location of that file?